Library/ViewModels/DiscoveryViewModel.swift

```diff
@@ -63,8 +67,18 @@ public protocol DiscoveryViewModelType {
 
 private func initialParam() -> DiscoveryParams {
 
+  let user = AppEnvironment.current.currentUser
+  let optedOutOfRecommendations = user?.optedOutOfRecommendations ?? true
+
+  if user == nil || optedOutOfRecommendations {
     return DiscoveryParams.defaults
       |> DiscoveryParams.lens.includePOTD .~ true
+  }
+
+  return DiscoveryParams.defaults
```

Comment:
If the user is not nil we return these defaults?

Reply:
We return these defaults if:
1- User is nil
2- User is not nil and ` user.optedOutOfRecommendations == true`